docs/validators/operation-mods.md

```diff
@@ -0,0 +1,61 @@
+---
+id: node-operation-modes
+title: Node operation modes
+---
+
+## Introduction
+
+Starting with `v1.4.x` release, a new CLI flag has been introduced to the node. It is `--operation-mode` and its purpose 
+is to override some configuration values that will allow the node to act differently, depending on the use-case.
+
+## List of available operation modes
+
+Below you can find a list of operation modes that are supported:
+
+### Full archive
+
+Usage:
+```
+./node --operation-mode full-archive
+```
+
+The `full-archive` operation mode will change the node's configuration in order to make it able to sync from genesis and also
+be able to serve historical requests. 
+Syncing a node from genesis might take same time due to the network's full archive peers count that generally isn't that big. 
```

Comment:
`since there aren't that many full archive peers to sync from` (optional)

Reply:
fixed